docs/guides/hooks/slack-notification/index.md

```diff
@@ -261,10 +263,83 @@ spec:
                 }
               ]
           }
-  retentionPolicy:
-    name: 'keep-last-5'
-    keepLast: 5
-    prune: true
+  executor:
+    type: Operator
+```
+
```

Comment:
Notice the `action` section. This hook sends an HTTP POST request. In the `path` field of the `httpPost` section, we have specified the Slack incoming webhook path that we copied in the last step of configuring Slack incoming webhook.